sktime/forecasting/reconcile.py

```diff
@@ -0,0 +1,355 @@
+# -*- coding: utf-8 -*-
+# !/usr/bin/env python3 -u
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements reconciled forecasters for hierarchical data."""
+
+__all__ = ["ReconcilerForecaster"]
+__author__ = [
+    "ciaran-g",
+]
+
+# todo: include the reconciler transformers?
```

Comment:
that's not a silly idea at all!

You could import the transformer, and if a string for the "transformer-like" reconciliation is given, you could just do `Reconciler(strategy) * forecaster` inside. (i.e., I would not copy-paste code but wrap the transformer, optionally depending on strategy)

Reply:
Ok cool 👍